### PR TITLE
Stripe

### PR DESCRIPTION
**Pull request by ValeriaRbk**
* fix basket amount verification
* fix transactions with no comment

### CODE REVIEW

**Review by AntonParker18 — APPROVED** [review]

**Review by Bogd1y — APPROVED** [review]

File: controllers/payment.controllers.js [hunk 1]
```diff
@@ -92,12 +92,14 @@ async function newSetupIntent(req, res, next) {
...
     });
   } catch (err) {
+    logger.error(err)
     const error = new HttpError(
       'Could not save card. Please try again later',
       500,
     );
     return next(error);
   }
+  logger.info('The card was added successfully')
```
Comment (KMyroslav) [inline controllers/payment.controllers.js]: Could you please add info about which user created a card?

File: controllers/payment.controllers.js [hunk 2]
```diff
@@ -461,14 +461,14 @@ async function createConnectedAccount(req, res, next) {
...
     });
-    console.log(accountLink);
   } catch (err) {
     const error = new HttpError(
       'Could not register you now. Please try again later.',
       500,
     );
     return next(error);
   }
+  logger.info('connected account created successfuly')
```
Comment (KMyroslav) [inline controllers/payment.controllers.js]: And maybe here too?

File: controllers/payment.controllers.js (hunk 1 above)
Comment (ValeriaRbk) [reply controllers/payment.controllers.js:101]: Sorry that is not possible because we don't receive card information on the server, it goes to Stripe

Comment (ValeriaRbk) [reply controllers/payment.controllers.js]: Done, check please

File: controllers/payment.controllers.js (hunk 2 above)
Comment (ValeriaRbk) [reply controllers/payment.controllers.js]: Aslo done

**Review by KMyroslav — APPROVED** [review]

**Pull request merged**